src/Fixtures/config.js

```diff
@@ -1,6 +1,7 @@
 export const DrillTypes = {
   FITNESS: 'fitness',
   FRISBEE: 'frisbee',
+  FRISBEEJUNIOR: 'frisbeejunior',
```

Comment:
I think it should be a separate configuration, such as `AgeCatefory` with values `junior` or `senior` ?

(on fitness programs we use the EquipmentLabel to filter)

Reply:
Good idea

Reply:
I've tried to create a filter by age category but I couldn't make it working so far